virttest/utils_test/__init__.py

```diff
@@ -2410,26 +2420,189 @@ def unload_stress(stress_type, params, vms=None, remote_server=False):
+        vms/baremetal hosts.
+
+        :param params: test params for stress tools
+        :return: Based on result of test would return True or False
+        :raise: TestError: raised if unable load stress tool or given input is incorrect
+        """
+
+        error = False
+        for server_vm, client_vm in self.pair_vms:
+            try:
+                params['stress_cmds_%s' % self.stress_type] = self.server_cmd
+                stress_server = VMStress(server_vm, self.stress_type, params)
+                params['server_pwd'] = params.get("password")
+                # wait so that guests get ip address, else get_address will
+                # fail
+                client_vm.wait_for_login().close()
+                server_vm.wait_for_login().close()
+                if self.iptables_rule:
+                    for vm_ip in [server_vm.get_address(), client_vm.get_address()]:
+                        params['server_ip'] = vm_ip
+                        Iptables.setup_or_cleanup_iptables_rules(
+                            [self.iptables_rule], params=params, cleanup=False)
+                if not stress_server.app_running():
+                    stress_server.load_stress_tool()
+                if self.need_profile:
+                    profile_backup = self.profile + '.backup'
+                    shutil.copy(self.profile, profile_backup)
+                    self.pat_repl.update(
+                        {"serverip": str(server_vm.get_address())})
+                    self.prepare_profile(self.profile, self.pat_repl)
+                    client_vm.copy_files_to(self.profile, "/home", timeout=60)
+                    shutil.copy(profile_backup, self.profile)
+                    os.remove(profile_backup)
+                else:
+                    self.client_cmd = self.client_cmd.format(
+                        str(server_vm.get_address()))
+                params['stress_cmds_%s' % self.stress_type] = self.client_cmd
+                stress_client = VMStress(client_vm, self.stress_type, params)
+                if not stress_client.app_running():
+                    stress_client.load_stress_tool()
+            except exceptions.TestError as err_msg:
+                error = True
+                logging.error(err_msg)
+        return error
+
+    def verify_unload_stress(self, params):
+        """
+        This function will:
+        1. verify if the VMs are reachable after the stress tests
+        2. unloads stress in multiVMs
+        """
+        error = False
+        for vm in self.vms:
+            try:
+                s_ping, o_ping = utils_net.ping(
+                    vm.get_address(), count=10, timeout=20)
+                logging.info(o_ping)
+                if s_ping != 0:
+                    error = True
+                    logging.error(
+                        "%s seem to have gone out of network", vm.name)
+                else:
+                    stress = VMStress(vm, self.stress_type, params)
```

Comment:
we can reuse the object created during `load_stress_tool()` using variable similar to  `self.stress_client_object` and use it to `unload_stress()`